### DISCUSSION TITLE
When do you expect to support the search API?

### DISCUSSION THREAD

**Discussion by meetgeekai** [mariosimao/notion-sdk-php#177] [General]
Great work on the extension. 
When do you expect to support the search API?

**Comment by mariosimao** [comment]
Hi @meetgeekai, thanks for using the SDK!

Search support is on the roadmap (#62). It will be my next priority, so I expect to release it in 2 weeks.

**Reply by mariosimao** [reply]
@meetgeekai, I have just released [version 1.5.0](https://github.com/mariosimao/notion-sdk-php/releases/tag/v1.5.0), which includes support for the search API.
